Sources/RxComposableArchitecture/SwiftUI/WithViewStore.swift

```diff
@@ -0,0 +1,697 @@
+    ///   var body: some View {
+    ///     WithViewStore(self.store, observe: \.selectedTab) { viewStore in
+    ///       TabView(selection: viewStore.binding(send: AppFeature.Action.tabSelected) {
+    ///         ActivityView(
+    ///           store: self.store.scope(state: \.activity, action: AppFeature.Action.activity)
+    ///         )
+    ///         .tag(AppFeature.Tab.activity)
+    ///         SearchView(
+    ///           store: self.store.scope(state: \.search, action: AppFeature.Action.search)
+    ///         )
+    ///         .tag(AppFeature.Tab.search)
+    ///         ProfileView(
+    ///           store: self.store.scope(state: \.profile, action: AppFeature.Action.profile)
+    ///         )
+    ///         .tag(AppFeature.Tab.profile)
+    ///       }
+    ///     }
+    ///   }
+    /// }
+    /// ```
+    ///
+    /// To read more about this performance technique, read the <doc:Performance> article.
+    ///
+    /// - Parameters:
+    ///   - store: A store.
+    ///   - toViewState: A function that transforms store state into observable view state. All
+    ///   changes to the view state will cause the `WithViewStore` to re-compute its view.
+    ///   - isDuplicate: A function to determine when two `ViewState` values are equal. When values
+    ///     are equal, repeat view computations are removed.
+    ///   - content: A function that can generate content from a view store.
+    public init<State>(
+        _ store: Store<State, ViewAction>,
+        observe toViewState: @escaping (_ state: State) -> ViewState,
+        removeDuplicates isDuplicate: @escaping (_ lhs: ViewState, _ rhs: ViewState) -> Bool,
+        @ViewBuilder content: @escaping (_ viewStore: ViewStore<ViewState, ViewAction>) -> Content,
+        file: StaticString = #fileID,
+        line: UInt = #line
+    ) {
+        self.init(
+            store: store.scope(state: toViewState, action: { $0 }),
+            removeDuplicates: isDuplicate,
+            content: content,
+            file: file,
+            line: line
+        )
+    }
+    
+    /// Initializes a structure that transforms a store into an observable view store in order to
+    /// compute views from store state.
+    ///
+    /// > Warning: This initializer is deprecated. Use
+    /// ``WithViewStore/init(_:observe:removeDuplicates:content:file:line:)`` to make state
+    /// observation explicit.
+    /// >
+    /// > When using ``WithViewStore`` you should take care to observe only the pieces of state that
+    /// your view needs to do its job, especially towards the root of the application. See
+    /// <doc:Performance> for more details.
+    ///
+    /// - Parameters:
+    ///   - store: A store.
+    ///   - isDuplicate: A function to determine when two `ViewState` values are equal. When values
+    ///     are equal, repeat view computations are removed.
+    ///   - content: A function that can generate content from a view store.
+    @available(
+        iOS,
```

Comment:
I think we can remove all deprecated things here, as this is new for RxTCA, wdyt?

Reply:
here [5983021](https://github.com/tokopedia/RxComposableArchitecture/pull/96/commits/59830215f3a87e54b65097e2765a2281c0b29086)